docs/csharp/programming-guide/concepts/linq/snippets/set-operators/Program.cs

```diff
@@ -0,0 +1,6 @@
+﻿using SolarSystem;
+
+Program.DistinctByExample();
+Program.ExceptByExample();
+Program.IntersectByExample();
+Program.UnionByExample();
```

Comment:
These four statements could move into the Program.cs file in the parent folder. (Bonus points if you want to update that program to use top level statements).